modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,86 @@
+= Introduction to data migration
+:page-tag: migration,zdm,zero-downtime,zdm-proxy,introduction
+ifdef::env-github,env-browser,env-vscode[:imagesprefix: ../images/]
+ifndef::env-github,env-browser,env-vscode[:imagesprefix: ]
+
+Enterprises today want to reliably migrate mission-critical client applications and data to cloud environments with zero downtime or near zero downtime during the migration.
+
+A migration is a workflow that encompasses the lifecycle of uploadng and importing your data to the selected keyspaces. 
+At {company}, we've developed a set of thoroughly-tested self-service tools to walk you through well-defined migration options:
+
+* Zero Downtime Migration
+* Cassandra Data Migrations
+* DSBulk Migration
+
+First, a couple of key terms to understand for using our migration tools:
+
+* **Origin:** This cluster is your existing Cassandra-based environment, whether it's open-source Apache Cassandra, DSE, or {astra_db} Classic.
+
+* **Target:** This cluster is the new environment to which you want to migrate client applications and data.
+
+For additional terms, see the xref:glossary.adoc[glossary].
+
+////
+[TIP]
+====
+An important migration prerequisite is that you already have the matching schema on Target. 
+A CQL statement that your client application sends to {zdm-proxy} must be able to succeed on both Origin and Target clusters. 
+This means that any keyspace that your client application uses must exist on both Origin and Target with the same name. 
+Table names must also match. For more, see xref:feasibility-checklists.adoc#_schemakeyspace_compatibility[Schema/keyspace compatibility].
+====
+
+Your migration project occurs through a sequence of phases. 
+Here is an example of these phases with ZDM using ZDM proxy, which is a simple and lightweight proxy that handles all the real-time requests generated by your client applications.
+For more, see http:addlinkhere.adoc[ZDM proxy]
+
+image::{imagesprefix}zdm-migration-all.png[Migration phases from start to finish]
+
+Before walking through illustrations of each phase, look at a pre-migration, high-level view. At this point, your client applications are performing read/write operations with an existing CQL-compatible database. That is, Apache Cassandra, DSE, or Astra DB.  
+
+image::{imagesprefix}zdm-migration-before-starting.png[Diagram shows existing CQL-compatible environment before migration starts.]
+
+Before any migration begins, complete prerequisites, prepare your environment, and set up the recommended infrastructure.
+////
+
+== Migrations with and without zero downtime examples
+
+Data migrations with downtime are those where the data being migrated is critical for ongoing operations, and the migration process itself requires systems to be taken offline temporarily. Downtime may be required for:
```

Comment:
Suggest breaking the first sentence into two shorter sentences and making the text more straightforward. For example, "Downtime may be necessary when you need to migrate data that is critical for ongoing operations. In some cases, the migration process itself requires systems to be taken offline temporarily."